go/internal/configschema/schema.go

```
@@ -0,0 +1,1209 @@
+// deleting or replacing operator intent.
+func validateStrictWriteFields(v *value) error {
+	if x := v.find("hostname"); x != nil && (x.kind != stringKind || strings.TrimSpace(x.text) == "") {
+		return errors.New("schema_invalid: hostname: must be a nonblank bind address")
+	}
+	if x := v.find("appOwnedMemoryBudgetMb"); x != nil && !validIntRange(x, 64, maxAppOwnedMemoryBudgetMB) {
+		return fmt.Errorf("schema_invalid: appOwnedMemoryBudgetMb: must be an integer from 64 to %d", maxAppOwnedMemoryBudgetMB)
+	}
+	if x := v.find("upstreamHostCircuitThreshold"); x != nil && !validIntRange(x, 0, 100) {
+		return errors.New("schema_invalid: upstreamHostCircuitThreshold: must be an integer from 0 to 100")
+	}
+	if x := v.find("googleAntigravityStaticCatalogVersion"); x != nil && !(x.kind == numberKind && (x.number.String() == "1" || x.number.String() == "2")) {
```

golangci-lint (2.13.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)